### PR TITLE
[MIG][14.0] account_banking_mandate

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by StefanRijnhart** [comment]
Thanks for all your work in these module ports again! Can I take the opportunity of asking what the use is of having the mandate_id on the move line as well as on the move, especially if the move's mandate_id is propagated to the move line's mandate ids. Would it make more sense not to propagate them on `_post` but instead use the one on the move line if it is set with a fallback on the mandate on the move if that one is set?

**Comment by alexis-via** [comment]
@StefanRijnhart I agee with you, it's strange to have mandate_id on both move and move.line. I think we should have it on the move only, and re-organise the code to read it there. I'll do that in the coming days/weeks.

**Comment by pedrobaeza** [comment]
Can you please check Travis?

**Comment by joao-p-marques** [comment]
I think Travis was failing because `account_payment_order` was not migrated yet. It was merged 2 days ago, so re-running the CI should unblock it @pedrobaeza

**Comment by pedrobaeza** [comment]
Yes, restarted and now green. Please review.

**Comment by StefanRijnhart** [comment]
I can file a PR into this one for my suggestion if needed.

File: account_banking_mandate/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
-# Copyright 2014 Compassion CH - Cyril Sester <csester@compassion.ch>
-# Copyright 2014 Tecnativa - Pedro M. Baeza
-# Copyright 2015-16 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
-# Copyright 2017 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2014-2020 Compassion CH - Cyril Sester <csester@compassion.ch>
+# Copyright 2014-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2015-2020 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2017-2020 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
```
Comment (joao-p-marques) [inline account_banking_mandate/__manifest__.py:4]: You do you change the Copyright from the previous changes? This should be left as is, and you should only add your Copyright with the current year

File: account_banking_mandate/models/account_banking_mandate.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# Copyright 2014 Compassion CH - Cyril Sester <csester@compassion.ch>
-# Copyright 2014 Tecnativa - Pedro M. Baeza
-# Copyright 2015-16 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2014-2020 Compassion CH - Cyril Sester <csester@compassion.ch>
+# Copyright 2014-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2015-2020 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
```
Comment (joao-p-marques) [inline account_banking_mandate/models/account_banking_mandate.py:3]: Same here

**Review by joao-p-marques — COMMENTED** [review]
@alexis-via could you squash the two migration commits into one?

Also, please check the comments bellow.

Otherwise, looks good :+1:

**Comment by pedrobaeza** [comment]
@StefanRijnhart that would be good for finishing this.

**Comment by StefanRijnhart** [comment]
I've created a draft https://github.com/OCA/bank-payment/pull/770. I've processed  @joao-p-marques's comments so we can use it as a superseding PR unless @alexis-via disagrees.

File: account_banking_mandate/README.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+This module depends on :
+* account_payment
+
+This module is part of the OCA/bank-payment suite.
+
+Usage
+=====
+
+To use this module, see menu "Invoicing/Accounting > Payments > Debit Orders"
```
Comment (CarlosRoca13) [inline account_banking_mandate/README.rst:53]:
```suggestion
To use this module, see menu "Invoicing/Accounting > Customers > Debit Orders"
```

**Comment by joao-p-marques** [comment]
I think we can close this in favour of #770, as @alexis-via seems unresponsive. @pedrobaeza can you do it?

**Pull request closed without merging**